### PR TITLE
Stats page from Livia!

### PR DESCRIPTION
**Pull request by ChuyueSun**
Stats page from Livia!

### CODE REVIEW

**Review by tsalako — CHANGES_REQUESTED** [review]
Hey Livia. I think you forgot to include some files in this push. Currently there is only Datastore.java. Also I dont think you need to make that index.html change.

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
     <h1>CodeU Starter Project</h1>
     <p>This is the CodeU starter project. Click the links above to login and visit your page.
     You can post messages on your page, and you can visit other user pages if you have
-    their URL.</p>
+    their URL. Hello World!</p>
```
Comment (tsalako) [inline src/main/webapp/index.html]: Can you remove this "Hello World" string.

**Review by tsalako — CHANGES_REQUESTED** [review]
This looks great. I only have a few comments.

1. Can you format these files so they are more readable (less extra lines, etc.). IJ probably has some built in formatter.
2. Can you include a screenshot of what you built.

File: src/main/webapp/index.html (hunk 1 above)
Comment (Jussepen) [reply src/main/webapp/index.html]: this looks good, other than the thing above. I compared to the one i have and the one on the website. ! good job !

**Review by Jussepen — APPROVED** [review]

**Review by tsalako — APPROVED** [review]
Can you include a screenshot?

**Comment by ChuyueSun** [comment]
Screen Shot 2019-03-13 at 8 49 36 PM

**Pull request merged**